### PR TITLE
[scatter] add option for text labels

### PR DESCRIPTION
**Pull request by beets**
v0 - no logic to control for text overlap
also added all the display options to a single row

image

### CODE REVIEW

File: static/js/tools/scatter/context.ts [hunk 1]
```diff
@@ -164,8 +166,9 @@ const FieldToAbbreviation = {
   lowerBound: "lb",
   upperBound: "ub",
 
-  // Quadrant fields
+  // DisplayOptions fields
   isQuadrant: "qd",
```
Comment (chejennifer) [inline static/js/tools/scatter/context.ts:170]: nit: maybe these can also be showQuandrant and showLabels

File: static/js/tools/scatter/context.ts [hunk 2]
```diff
@@ -176,6 +179,7 @@ function useContextStore(): ContextType {
   const [y, setY] = useState(EmptyAxis);
   const [place, setPlace] = useState(EmptyPlace);
   const [isQuadrants, setQuadrants] = useState(false);
```
Comment (chejennifer) [inline static/js/tools/scatter/context.ts:181]: nit: maybe change to showQuadrants and showLabels to be consistent?

**Review by chejennifer — APPROVED** [review]
nice!

File: static/js/tools/scatter/context.ts (hunk 1 above)
Comment (beets) [reply static/js/tools/scatter/context.ts]: done!

File: static/js/tools/scatter/context.ts (hunk 2 above)
Comment (beets) [reply static/js/tools/scatter/context.ts]: done!

**Pull request merged**